database/things/mysql.pldb

```diff
@@ -62,6 +62,7 @@ rijuRepl https://riju.codes/mysql
  example
   SELECT 'Hello, world!';
   
+
```

Comment:
lets see if we can remove these and keep the PRs really focused on the meaningful changes

Reply:
@breck7 , these changes are the result of npm run format so I pushed these too.

Reply:
Got it! No problem then.